lib/ood_core/job/adapters/sge/qstat_xml_r_listener.rb

```diff
@@ -24,21 +24,32 @@ def initialize
     @parsed_jobs = []
     @current_job = {
       :tasks => [],
-      :native => {}  # TODO: improve native reporting
+      :native => {
+        :ST_name => ''
+      }  # TODO: improve native reporting
     }
     @current_text = nil
+    @stdout_PN_path = false
 
     @current_request = nil
+    @native_tags = ['JB_job_number', 'JB_job_name', 'JB_version', 'JB_project', 'JB_exec_file', 'JB_script_file', 'JB_script_size', 'JB_submission_time', 'JB_execution_time', 'JB_deadline', 'JB_owner', 'JB_uid', 'JB_group', 'JB_gid', 'JB_account', 'JB_cwd', 'JB_notify', 'JB_type', 'JB_reserve', 'JB_priority', 'JB_jobshare', 'JB_verify', 'JB_checkpoint_attr', 'JB_checkpoint_interval', 'JB_restart']
```

Comment:
Some of these are already accounted for, like `JB_job_number`, `JB_job_name`. But even more so like `JB_submission_time` because in my PR we can't just extract it, we need to post process it.

Reply:
I see, I guess it's just additive in native. It'll add to native, but _also_ add it to the common script attribute.